main.js

```diff
@@ -31,12 +64,40 @@ function navList() {
   nav.style.display = 'block';
 }
 
+function popList() {
+  view.classList.add('pop-up-menu');
+  view.style.display = 'block';
+  flow.style.overflow = 'hidden';
+  background.style.opacity = '0.5';
+  background.style.filter = 'blur(6px)';
+
+  // for name, desciption, link, image, and technologies
+  // set their innerText with the first object from the array, i.e
+  // const project = projects[0]
+  // name.innerText =  project.name
```

Comment:
- [OPTIONAL] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.